src/OSLikeStuff/task_scheduler.cpp

```diff
@@ -472,7 +472,7 @@ void TaskManager::resetStats() {
 }
 
 void TaskManager::printStats() {
-	D_PRINTLN("Dumping task manager stats: (min/ average/ max)");
```

Comment:
can we keep this?  it's helpful for finding issues since it can be grabbed by users using the midi logging